### PR TITLE
Initial Telos changes + removing other chains

### PR DESCRIPTION
**Pull request by jokbal-yum**
- Removing all chains + chain data + ABI + stats
- Adding Telos address-book information
- Adding Telos LP pool information for 3 LP's on Omnidex Finance
- Adding Telos Vault and Strategy information for USDC/USDT farm
CleanShot 2022-03-24 at 23 03 41@2x

CleanShot 2022-03-24 at 22 55 21@2x
CleanShot 2022-03-24 at 22 55 30@2x

### CODE REVIEW

File: src/utils/getVaults.js [hunk 1]
```diff
@@ -2,11 +2,11 @@ const axios = require('axios');
 
 const getVaults = async vaultsEndpoint => {
   try {
-    const response = await axios.get(vaultsEndpoint);
-    const data = response.data;
-    let vaults = '[' + data.substring(data.indexOf('\n') + 1);
-    vaults = eval(vaults);
-    return vaults;
+    // const response = await axios.get(vaultsEndpoint);
```
Comment (payolaplumber) [inline src/utils/getVaults.js:5]: What's the purpose of this change?

File: src/api/stats/telos/getOmnidexLpApys.js [hunk 2]
```diff
@@ -1,62 +1,85 @@
...
+  console.log("Alloc Points: ", allocPoints);
+
   const pairAddresses = pools.map(pool => pool.address);
-  const tradingAprs = await getTradingFeeAprSushi(joeClient, pairAddresses, liquidityProviderFee);
 
   for (let i = 0; i < pools.length; i++) {
+
     const pool = pools[i];
 
+    console.log("Pool: ", pool);
```
Comment (payolaplumber) [inline src/api/stats/telos/getOmnidexLpApys.js]: Do we need these log still? Prefer at a minimum fewer logs so that they're less spammy in prod.

**Review by payolaplumber — APPROVED** [review]
LGTM with a couple nits

File: src/api/stats/telos/getOmnidexLpApys.js (hunk 2 above)
Comment (jokbal-yum) [reply src/api/stats/telos/getOmnidexLpApys.js]: These were for debugging, let me take them out, forgot to do that

File: src/utils/getVaults.js (hunk 1 above)
Comment (jokbal-yum) [reply src/utils/getVaults.js:5]: ts supposed to pull the data from the github link like this:

https://raw.githubusercontent.com/beefyfinance/beefy-app/prod/src/features/configure/vault/moonbeam_pools.js

so once that code is in the app part and its public, we can include the url and it'll pull the data from there. currently i'm pulling it from a hardcoded version of that in the api repo but ideally it'd be like above so i was saving that code for when we switch back
